releasenotes/notes/ato_sdk_v2-35c17dc258c5b690.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    AAP: This introduces a new user event sdk available through ddtrace.appsec.track_user_sdk for manual instrumentation.
+    More information on our documentation page.
```

Comment:
I think it is worth adding a link to that specific documentation page since we have tons of docs.

Reply:
This is still work in progress...